core/framework/extractors/latex_extractor_backup_20250913.py

```
@@ -0,0 +1,494 @@
+    
+    def _convert_to_markdown(self, tex_path: Path) -> Tuple[Optional[str], Optional[str]]:
+        """
+        Convert LaTeX to markdown using pandoc if available.
+        
+        Returns:
+            Tuple of (markdown_text, error_message)
+        """
+        self.pandoc_stats['attempts'] += 1
+        try:
+            import subprocess
+            result = subprocess.run(
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)